src/core/lib/surface/channel_init.h

```diff
@@ -24,6 +24,7 @@
 #include <functional>
 #include <vector>
 
+#include "src/core/lib/channel/channel_stack_builder.h"
 #include "src/core/lib/surface/channel_stack_type.h"
```

Comment:
Looks like this include is no longer needed.

Reply:
fixed